Assignment3/test.js

```diff
@@ -0,0 +1,64 @@
+var array = [];
+array[0] = [3,56,7,32,9,14];
+array[1] = [21, 87, 91, 31, 97, 59, 8, 83, 27, 23, 76, 72, 31, 91, 65, 77, 84, 27, 56, 59];
+array[2] = [21, 87, 91, 31, 97, 59, 8, 83, 27, 23, 76, 72, 31, 91, 65, 77, 84, 92, 56, 59];
+array[3] = [111, 0, 20, 0, 2];
+//var val = [32, 91, 92, 20];   // Results for second largest
+
+var string = [];
+string[0] = "dummy";
+string[1] = "dummy text";
+string[2] = "shali is running fast";
+string[3] = "u@#d$fkj$#!kds";
```

Comment:
why write 5 statements for declaring an array of 5 strings? declare in a single statement instead.